deployment/grid/terraform/compute_plane/data.tf

```diff
@@ -3,18 +3,44 @@
 # Licensed under the Apache License, Version 2.0 https://aws.amazon.com/apache-2-0/
 
 
+# These resources are used to give AWS Load Balancer Controller time to create
+# and delete the external AWS resources, i.e. ALB, NLB and Target Groups
+resource "time_sleep" "influxdb_service_dependency" {
```

Comment:
Is this one required:
influxdb_service  load balancer should not rely AWS ALB Load balancer controller because it is not associated to any ingress resource
What do you think ?